### PR TITLE
SVG rendering

### PR DESCRIPTION
**Pull request by werthdavid**
Made some changes not related to JS code. If you consider merging this, I can update the PR so only the changes relevant for SVG rendering will be merged.

### CODE REVIEW

**Comment by rugk** [comment]
Fixes https://github.com/lrsjng/kjua/issues/7

You can (automatically) [let issues close](https://help.github.com/articles/closing-issues-via-commit-messages/) when a PR is merged.

File: README.md [hunk 1]
```diff
@@ -43,14 +67,20 @@ Click [here](https://werthdavid.github.io/kjua/)
...
 * `image` additional image (defaults to `undefined`, use an HTMLImageElement or base64-string)
+* `imageAsCode` draw the image as part of the code (defaults to `false`)
 
 More details can be found on [larsjung.de/kjua](https://larsjung.de/kjua/)
 
 ## Differences to kjua
 
-This is basically just a fork of kjua that adds the possibility to render QR-codes as SVG.
+* possibility to render QR-codes as SVG
+* transparent background support
```
Comment (rugk) [inline README.md:85]: And how can we use that?

Comment (werthdavid) [reply README.md:85]: Transparent background? Just provide null/undefined or an empty string as „back“ option

**Comment by lrsjng** [comment]
Added a svg mode. I think the lib you use is way to heavy for a simply svg mode addition, makes the dist size ~6x as heavy.

**Comment by rugk** [comment]
Hmm, seems to be done in https://github.com/lrsjng/kjua/commit/3b8216ea1f109a69713df4b8c3afeb87ccff00ee.
(Also maybe it would be useful to use proper commit messages, so they are any useful. :thinking:)

@werthdavid do you maybe want to comment on that? If upstream now has SVG support #7 is basically solved… So why should one use your fork? (Are there other advantages that justify that additional size?) :thinking:

**Comment by werthdavid** [comment]
There's a hand full of differences in my fork but the basic idea was the svg rendering. now that @lrsjng is "back" we can also close my forked version. Would you be interested in those changes in the original kjua? I could make a PR then.

**Pull request closed without merging**